### ISSUE TITLE
TypeScript error occurs when `--strictNullChecks`disabled

### ISSUE THREAD

**Issue by laudaikinhdi** [Quramy/prisma-fabbrica#307]
<img width="1213" alt="image" src="https://github.com/Quramy/prisma-fabbrica/assets/34116228/cb65483f-441a-4552-a2c5-3afaaa409e63">

**Comment by tak-mitani** [comment]
We have the same issue.

**Comment by Quramy** [comment]
I'm not familiar with Nest.js and I don't know how to reproduce this TS compilation error.

I think it's related to your project's TypeScript compiler options (but I am uncertain).

1. Would you paste tsconfig.json and schema.prisma file reproducing this ?
2. It seems that `noTranspile` option is enabled. Does this error keep without `noTranspile` option ?

**Comment by laudaikinhdi** [comment]
Hi @Quramy 
schema.prisma
```prisma
generator fabbrica {
  provider    = "prisma-fabbrica"
  output      = "../node_modules/prisma-fabbrica"
  tsconfig    = "../tsconfig.json"
  noTranspile = false
}
```
tsconfig.json
```ts
{
  "$schema": "https://json.schemastore.org/tsconfig",
  "extends": "./base.json",
  "compilerOptions": {
    "module": "commonjs",
    "declaration": true,
    "removeComments": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "allowSyntheticDefaultImports": true,
    "target": "ES2021",
    "sourceMap": true,
    "incremental": true,
    "skipLibCheck": true,
    "strictNullChecks": false,
    "noImplicitAny": false,
    "strictBindCallApply": false,
    "forceConsistentCasingInFileNames": false,
    "noFallthroughCasesInSwitch": false
  }
}
```
I see just need remove `?? {}` under picture. Its okay
<img width="631" alt="image" src="https://github.com/Quramy/prisma-fabbrica/assets/34116228/d6192c4b-fa37-47bd-bebf-4c46d83d5838">

**Comment by Quramy** [comment]
@laudaikinhdi 

Thanks for reply. 

I can reproduce this issue and I found `"strictNullChecks": false` causes the compilation error ( It seems a little odd). Either way I'll fix this issue.

**Comment by Quramy** [comment]
I fixed this via #341 and released v2.1.5. Pls check it out.
